assets/console-plugin/config-map.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+data:
+  nginx.conf: |
+    error_log /dev/stdout info;
+    events {}
+    http {
+      access_log         /dev/stdout;
```

Comment:
Is there a risk of excessive logging? Maybe we do not need every access to be logged, we just need to log exceptional access such as errors.
Here is conditional logging config: https://docs.nginx.com/nginx/admin-guide/monitoring/logging/#enabling-conditional-logging

Reply:
I am not sure what is the right amount of logging that the new plugin should have.  This is based on the helm chart for plugin. 

@kyoto any thoughts?

Reply:
I have removed the access logs.